### ISSUE TITLE
Support the Web Content Accessibility Guidelines (WCAG)

### ISSUE THREAD

**Issue by mgesing** [amzn/amazon-pay-api-sdk-dotnet#20]
Currently, the buttons (payment and login) violate the guideline for [alternative text for images](https://dequeuniversity.com/rules/axe/4.10/image-alt).

Accessibility warning in Chrome's Lighthouse:

![Image](https://github.com/user-attachments/assets/353d6647-b02a-4b19-8440-d490c3f36acf)

**Comment by shangamesh** [comment]
Thanks for flagging it. This repo is primarily focus on API calls for payment transaction.
However we passed the feedback to internal team for fixing the accessibility issue in Amazon Pay button
